app/views/components/forms/_custom_layout.html.erb

```diff
@@ -3,7 +3,7 @@
     <div class="grid-col-12">
       <%= render "components/forms/logo_and_title", form: form %>
       <%- if form.instructions.present? %>
-        <div class="fba-instructions">
+        <div class="fba-instructions margin-bottom-2">
```

Comment:
this .fba-instructions was recently a <p>, which is styled to have its own margins
but now that its a <div> it needs explicit margins; thus adding space